src/legacy/ui/public/vislib/lib/axis/axis_scale.js

```diff
@@ -151,7 +151,15 @@ export class AxisScale {
     const domain = [min, max];
     if (this.axisConfig.isUserDefined()) return this.validateUserExtents(domain);
     if (this.axisConfig.isLogScale()) return this.logDomain(min, max);
-    if (this.axisConfig.isYExtents()) return domain;
+    if (this.axisConfig.isYExtents()) {
+      const scaleBottomMargin = this.axisConfig.get('scale.bottomMargin');
+      if (scaleBottomMargin === 0) {
+        return domain;
+      } else {
+        domain[0] = domain[0] - scaleBottomMargin;
```

Comment:
The `scaleBottomMargin` doesn't consider the case of a full negative set of data and a mixed negative/positive situation. It will be great if we can check that and adjust the domain accordingly.

Reply:
considering the negative and mixed (negative/positive evtl. with 0) -> see my comment below